### PR TITLE
Add pre commit hook

### PR DESCRIPTION
**Pull request by Tomekmularczyk**
RE #23 

Adds lint-staged which runs "lint" command before the commit, so that every commit is clean from eslint errors.

Also adds a note about Pull Requests to README.md. Not sure whether it looks the way you wanted @jhidalgoesp , let me know and I can change it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "node ./bin/www",
     "dev": "nodemon ./bin/www",
-    "lint": "eslint controllers models routes"
+    "lint": "eslint ."
```
Comment (Tomekmularczyk) [inline package.json]: I changed it to run on all files from directory root instead

Comment (jhidalgoesp) [reply package.json]: Nice.

**Comment by jhidalgoesp** [comment]
Looks great.  Can you please solve the conflicts so i can merge it?

**Comment by Tomekmularczyk** [comment]
@jhidalgoesp resolved conflicts in the PR

**Comment by jhidalgoesp** [comment]
Thanks for your help. Solved #25

**Pull request merged**